scripts/toolbox/modules/Feed.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import subprocess
+
+
+def start(cfg, operations):
```

Comment:
I think we should create the _work_ folder here if it does not exist. I got an exception first time I ran it. Worked fine after I created it manually.

Reply:
I see, will add it!

Reply:
Added